src/ItemCard.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import "./App.css";
+import bike from './images/bike2.jpg';
+
+const ItemCard = () => {
+  return (
+    <section className="itemcard">
+
+        <div className='itemimage'>
+        <img className='itemimage'src={bike} alt="" height='140px' width='200px' />
+        </div>
+        
+        <div className='itemtext'>
+          <h2> As New Racing Bike </h2>
+          <h4> 30 Points </h4>
+          <p>Available for 3 weeks</p>
+          <p className='blue'>2400 NV, Copenhagen  </p>
+        </div>
```

Comment:
You shouldn't have hardcoded values at this point. Instead use your database and populate item list with real data